src/components/TodoList/TodoList.jsx

```diff
@@ -1,44 +1,117 @@
 import React from 'react';
 import './TodoList.scss';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
 
-export const TodoList = () => (
-  <div className="TodoList">
-    <h2>Todos:</h2>
+export class TodoList extends React.Component {
 
-    <div className="TodoList__list-container">
-      <ul className="TodoList__list">
-        <li className="TodoList__item TodoList__item--unchecked">
+  state = {
+    query: '',
+    select: 'all',
+  };
+
+
+  handleChange = (event) => {
+    const {name, value} = event.target;
+
+    this.setState({ [name]: value });
+  };
+
+
+  render() {
+    const { query,select } = this.state;
+    const { todos, selectUser } = this.props;
+
+    const filterTasks = todos
+    .filter( task => {
+      if(query) {
+        return (
+          task.title !== null
+          && task.title.toLowerCase().includes(query.toLowerCase())
+        )
+      }
+      return task;
+    })
+
+    .filter(task => {
+      if( select === 'Active') {
+        return task.completed === false;
+      } else if ( select === 'Completed') {
+        return task.completed === true;
+      }
+      return task;
+    });
```

Comment:
nice solution, but's let's move the functions inside filter methods to the separate functions - filterByStatus and filterByQuery, so then we can pass them like this

`const filteredTasks = todos.filter(this.filterByQuery).filter(this.filterByStatus);`